game.js

```diff
@@ -171,7 +186,61 @@ class Game extends Screen {
 	}
 
 	drawBackground() {
-		if (this.bgImage) background(this.bgImage, 100)
+		if (!this.bgImage) return;
```

Comment:
Try not to short-circuit functions by returning unnecessarily... it tends to disrupt the flow of how we read code and can make debugging more difficult.  Let it the function flow to the end when possible.

Reply:
Yes, exactly

Reply:
Resolved in 3189f8e6115534c7e4ae2cd245ea06fb6f3d8d68